src/components/css/index.css

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
The indentation for this file is a little off.

Typically should be uniform with this format:
```
body {
  font-family: Helvetica;
}
```

body should be followed by a space then the curly.
the style should be indented with two spaces and the style should be followed by a colon, a space, then the value.